src/groovy/au/org/emii/gogoduck/job/Job.groovy

```diff
@@ -37,6 +37,10 @@ class Job {
         new URL("$serverUrl/aggr/$uuid")
     }
 
+    URL getStatusUrl() {
+        new URL("$serverUrl/job/$uuid")
```

Comment:
I saw that. I'll try to explain the problem although it might not make sense to you because you've been developing on grails for a while.

I see the UrlMappings as a way to advise a controller what to show, but not really as an authoritative way of deciding which functions it really exposes. I will not lie - it took me a bit to actually find out that there is an entry in UrlMappings. In my personal opinion UrlMappings is a bit of a code smell - it can be used, but not relied on. Let me know if it doesn't make sense.